backend/src/cronJob.ts

```diff
@@ -0,0 +1,32 @@
+import cron from "node-cron";
+import { fetchMedia } from "./media";
+import { fetchTextMessagesFromChannels } from "./text";
+import { channels } from "..";
+
+cron.schedule("0 * * * *", async () => {
+  try {
+    console.log("Starting scheduled fetch at", new Date().toISOString());
+
+    // Fetch Memes
+    try {
+      const memeChannelIds = channels.memes;
+      for (const channelId of memeChannelIds) {
+        await fetchMedia(channelId, 10);
```

Comment:
Kanskje ikke sette limit på hvor mange den fetcher? Ikke sikkert det er mange tilfeller, men kan jo hende det er noe media som ikke blir med hvis det er over 10 i timen. Eneste tilfelle jeg kan se for meg at de ter aktuelt er under genfors da...

Reply:
Tror det er lurt, med en form for limit, med tanke på uforutsett spam eller lignende. Kan evnt øke limiten til 100?

Reply:
100 holder sikkert ja